app/screens/selector_screen/selected_options/selected_options.tsx

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {View} from 'react-native';
+
+import {ViewTypes} from '@constants';
+import {Channel} from '@mm-redux/types/channels';
+import {DialogOption} from '@mm-redux/types/integrations';
+import {Theme} from '@mm-redux/types/preferences';
+import {UserProfile} from '@mm-redux/types/users';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+import SelectedOption from './selected_option';
+
+type Props = {
+    theme: Theme;
+    selectedOptions: DialogOption[]|UserProfile[]|Channel[];
+    dataSource: string;
+    onRemove: (opt: DialogOption|UserProfile|Channel) => void;
+}
+
+export default function SelectedOptions(props: Props) {
```

Comment:
One big issue seems that this component does not limit the amount of selected items and thus the view can grow limitless taking space from the list that sits below it on the screen. For me this is a blocker, look into adding some sort of ScrollView with a max height to avoid having this component filling up the screen.